frontend/src/index.scss

```diff
@@ -2804,8 +2775,6 @@ $easeInOutCirc: cubic-bezier(0.785, 0.135, 0.150, 0.860);
     background: rgba(200, 205, 210, 0.16);
 
     max-height: 300px;
-    overflow: hidden;
```

Comment:
Removing this lines fixes an overflow in a Preview Fields modal. Before it created a scroll element, now it relies on an extendability of a modal and plays much better with overlays.